### ISSUE TITLE
[rbac-manager] update chart to use non-deprecated api version for CRD 

### ISSUE THREAD

**Issue by kimschles** [FairwindsOps/charts#284]
**Is your feature request related to a problem? Please describe.**
The CRD definition `apiextensions.k8s.io/v1beta1` is deprecated in Kubernetes 1.16, yet it is still in use by the rbac-manager helm chart. 

**Describe the solution you'd like**
Change `apiextensions.k8s.io/v1beta1` to  `apiextensions.k8s.io/v1` on the first line of the CRD.
https://github.com/FairwindsOps/charts/blob/master/stable/rbac-manager/templates/customresourcedefinition.yaml#L1

**Comment by sudermanjr** [comment]
Since the api is not removed until kube 1.19.0, and the new api is not present in 1.15, we are going to wait until a later time to fix this.

I am going to change this to a bug, and make sure we revisit soon.

**Comment by js-timbirkett** [comment]
Looks like this got fixed in #385 - don't forget that you can act programmatically based on supported API capabilities like:
```yaml
{{- if .Capabilities.APIVersions.Has "apiextensions.k8s.io/v1" }}
apiVersion: apiextensions.k8s.io/v1
{{- else }}
apiVersion: apiextensions.k8s.io/v1beta1
{{- end }}
```

Which makes upgrades like the Kubernetes 1.15 -> 1.16+ upgrade a bit more comfortable 😄
